libs/api/domains/financial-aid/src/lib/financialAid.resolver.ts

```diff
@@ -0,0 +1,28 @@
+import { Query, Resolver } from '@nestjs/graphql'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  IdsUserGuard,
+  ScopesGuard,
+  CurrentUser,
+} from '@island.is/auth-nest-tools'
+import { UseGuards } from '@nestjs/common'
+
+import { FinancialAidService } from './financialAid.service'
+import { CurrentApplicationResponse } from './dto/currentApplication'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver()
+export class FinancialAidResolver {
+  constructor(private financialAidService: FinancialAidService) {}
+
+  @Query(() => CurrentApplicationResponse, { nullable: true })
+  async hasUserFinancialAidApplicationForCurrentPeriod(
```

Comment:
Should be something like `{namespace}FinancialAidApplicationStatus` to better fit our naming conventions?

Or `{namespace}FinancialAidApplications` with optional filters like “period=CURRENT” or whatever.